### DISCUSSION TITLE
Code::Blocks IDE

### DISCUSSION THREAD

**Discussion by MichaelAgarkov** [catppuccin/catppuccin#2560] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

It is (was before wxWidgets 3.3) [quite tricky](https://forums.wxwidgets.org/viewtopic.php?t=34742) to theme it, but it is possible (e.g. [dark mode for wxMSW](https://github.com/adbrt/codeblocks-dark-mode-msw)).
I guess themes will all be in different conf files.

### Any additional comments?

Will this be looked at? It's my most favourite IDE because it works perfectly for me, and there aren't really any custom themes for it at the moment.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
> Will this be looked at? It's my most favourite IDE because it works perfectly for me, and there aren't really any custom themes for it at the moment.

Hey 👋,

Catppuccin is driven by community contributions so there is no guarantee that "this will be looked at" as you mention. We're more than happy to guide willing contributors in creating themes but the effort will need to come from someone such as yourself or another person who is interested in theming this IDE.
